config/public-api-v1.yml

```
@@ -31,31 +31,441 @@ paths:
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/SingleCourseResponse"
+                $ref: "#/components/schemas/CourseSingleResponse"
 components:
   schemas:
-    SingleCourseResponse:
+    CourseAttributes:
+      type: object
+      required:
+        - code
+      properties:
+        about_accredited_body:
+          type: string
+          format: markdown
+          description: Description of the accredited body for this course.
+          example: ""
+        about_course:
+          type: string
+          description: Short factual summary of the course formatted as Markdown.
```

```suggestion
          description: Short factual summary of the course.
```

Pretty sure the rest of the `description` fields don't bother with stating `formatted as Markdown`